packages/rust-bindings/src/aux_functions.rs

```diff
@@ -774,6 +774,12 @@ fn deserialize_type_value(
     }
 }
 
+pub fn display_type_schema_template_aux(schema: HexString) -> Result<JsonString> {
```

Comment:
I can see this is the same as it is done for other functions, so it is consistent, but I think we could just return the `Value` directly since wasm-bindgen can deal with those.